tests/integration/helpers/is-component-definition-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | is-component-definition', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
+  test('it renders', async function(assert) {
+    this.set('inputValue', '1234');
+
+    await render(hbs`{{is-component-definition inputValue}}`);
+
+    assert.equal(this.element.textContent.trim(), '1234');
```

Comment:
This test needs updating, surely it doesn't pass?

Reply:
many tests should be updated if we want to be consistent with latest ones...though this should work well (ember always has backwards support)